src/aspire/utils/filters.py

```diff
@@ -268,3 +269,47 @@ class RadialCTFFilter(CTFFilter):
     def __init__(self, pixel_size=10, voltage=200, defocus=15000, Cs=2.26, alpha=0.07, B=0):
         super().__init__(pixel_size=pixel_size, voltage=voltage, defocus_u=defocus, defocus_v=defocus, defocus_ang=0,
                          Cs=Cs, alpha=alpha, B=B)
+
+    def convert_from(self, filt):
+        """
+        Convert the non-radial CTF filter to a radial one.
+        """
+        defocus = np.sqrt(filt.defocus_u*filt.defocus_u + filt.defocus_v*filt.defocus_v)
+        self.pixel_size = filt.pixel_size
+        self.voltage = filt.voltage
+        self.wavelength = voltage_to_wavelength(self.voltage)
+        self.defocus_u = defocus
+        self.defocus_v = defocus
+        self.defocus_ang = 0
+        self.Cs = filt.Cs
+        self.alpha = filt.alpha
+        self.B = filt.B
+        self.defocus_mean = 0.5 * (self.defocus_u + self.defocus_v)
+        self.defocus_diff = 0.5 * (self.defocus_u - self.defocus_v)
+
+    def fb_mat(self, fbasis):
```

Comment:
This function looks like an almost exact copy of `radial_filter2fb_mat`. We can only keep one. Either this one calls that one or that one calls this one (or we remove the other one, since it's not used, I think).

Reply:
The fb_mat function have been re-implemented to call the `radial_filter2fb_mat` util function in commit 1693ad8.